src/main/java/com/iexec/worker/compute/pre/PreComputeService.java

```diff
@@ -141,35 +154,19 @@ public String runTeePreCompute(TaskDescription taskDescription, WorkerpoolAuthor
     private boolean prepareTeeInputData(TaskDescription taskDescription, String secureSessionId) {
         String chainTaskId = taskDescription.getChainTaskId();
         log.info("Preparing TEE input data [chainTaskId:{}]", chainTaskId);
-        // get image URI
-        PreComputeConfig preComputeConfig = smsService.getPreComputeConfiguration();
-        if (preComputeConfig == null
-                || StringUtils.isEmpty(preComputeConfig.getImage())
-                || StringUtils.isEmpty(preComputeConfig.getHeapSize())) {
-            log.error("Failed to get TEE pre-compute configuration from SMS " +
-                    "[chainTaskId:{}]", chainTaskId);
-            return false;
-        }
-        String preComputeImageUri = preComputeConfig.getImage();
-        // pull image
-        if (!dockerService.getClient().pullImage(preComputeImageUri)) {
```

Comment:
We probably need to keep it to be more resilient (same comment than above)?

Reply:
See the above comment.